AndroidManifest.xml

```diff
@@ -188,7 +188,7 @@
 
         <provider android:name="com.android.calendar.CalendarRecentSuggestionsProvider"
```

Comment:
I think we should also change the provider name to prevent conflicts with AOSP calendar.

Reply:
Ah I see. Yeah, I think I was wrong, since the name is not public, only the authorities.